Analysis/tasks.py

```diff
@@ -0,0 +1,152 @@
+import law
+import luigi
+import os
+import shutil
+import time
+import yaml
+from RunKit.sh_tools import sh_call
+from RunKit.checkRootFile import checkRootFileSafe
+
+from run_tools.law_customizations import Task, HTCondorWorkflow, copy_param, get_param_value
+from AnaProd.tasks import AnaTupleTask, DataMergeTask
+
+class HistProducerFileTask(Task, HTCondorWorkflow, law.LocalWorkflow):
+    max_runtime = copy_param(HTCondorWorkflow.max_runtime, 10.0)
+    n_cpus = copy_param(HTCondorWorkflow.n_cpus, 1)
```

Comment:
I set n_cpus to 1 for the moment, it can be changed. Max runtime can be downed also to 2 as far as I saw, and I'll do it. I'll keep 3 hrs to stay conservative